src/js/video.js

```diff
@@ -33,7 +33,9 @@ import xhr from 'xhr';
 import Tech from './tech/tech.js';
 
 // HTML5 Element Shim for IE8
-if (typeof HTMLVideoElement === 'undefined') {
+if (typeof HTMLVideoElement === 'undefined' &&
+    window.document &&
+    window.document.createElement) {
```

Comment:
What about using `document` here instead of `window.document`? We're already importing it on line 8.